kart/gui/diffviewer.py

```diff
@@ -333,8 +346,15 @@ def fillTree(self):
                         old = feat
                         new = {}
                     else:
-                        old = changes[f"U-::{featid}"]
-                        new = changes[f"U+::{featid}"]
+                        # switch on the type of format string that was found
+                        # TODO - remove support for 'old' format, requires users to have upgraded
+                        #  this plugin first
+                        if elementtype:
```

Comment:
Instead of adding a comment to explain `switch on the type of format string that was found`
consider giving this mode a variable with a name eg `is_new_id_format` - then you probably don't need the comment.
```
is_new_id_format = bool(elementtype)
if is_new_id_format:
  ...
else:
  ...
```